### DISCUSSION TITLE
Central de notificação

### DISCUSSION THREAD

**Discussion by kavalcante** [frontendbr/forum#1385] [Perguntas e Respostas]
Pessoal, aqui na empresa utilizamos notificação somente por e-mail. Porém, gostaríamos de notificar via push e SMS também.

Alguém conhece alguma plataforma SaaS que centralize as notificações? Não é o cliente que envia as notificações como o SendGrid, OneSignal ou Twillio.

O ideal seria um serviço que saberia notificar os usuários baseado no canal que eles pediram para serem notificados, um gerenciador de canais disponíveis mesmo.

Alguém conhece uma solução para isso?

**Comment by arthureden** [comment]
@kavalcante Conheço a Dito.
https://www.dito.com.br

**Comment by kavalcante** [comment]
> @kavalcante Conheço a Dito.
> https://www.dito.com.br

Obrigado pela resposta! Vou checar com mais calma.

Enquanto isso, eu achei o PinPoint da Amazon: https://aws.amazon.com/pt/pinpoint/

Conseguimos criar uma POC mínima com o OneSignal para servir da mesma maneira e vamos criar uma com o PinPoint também.

**Comment by tcelestino** [comment]
@kavalcante, ia sugerir a [ExactTarget](https://www.salesforce.com/products/marketing-cloud/overview/?mc=marketingcloud.com) (Salesforce), mas não sei qual é nivel de investimento a empresa que você trabalha quer gastar com isso.

**Comment by kavalcante** [comment]
Obrigado pela indicação também @tcelestino 

Respondendo sua pergunta, o ideal é sempre o melhor pelo menor preço hahahaha.
